Zend/zend_vm_def.h

```diff
@@ -9961,6 +10006,27 @@ ZEND_VM_HANDLER(209, ZEND_INIT_PARENT_PROPERTY_HOOK_CALL, CONST, UNUSED|NUM, NUM
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(211, ZEND_DEREF, VAR, UNUSED)
```

Comment:
This should be a cold handler. It's also possible we can completely drop it and replace with a `QM_ASSIGN`, which also unwraps.